test/service-sdk-tests/pom.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0"?>
+<!--
+  ~ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License").
+  ~ You may not use this file except in compliance with the License.
+  ~ A copy of the License is located at
+  ~
+  ~  http://aws.amazon.com/apache2.0
+  ~
+  ~ or in the "license" file accompanying this file. This file is distributed
+  ~ on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+  ~ express or implied. See the License for the specific language governing
+  ~ permissions and limitations under the License.
+  -->
+
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
```

Comment:
Should we move those tests to s3 instead? I'm not sure what service tests should to added this new module and not service modules

Reply:
In the `s3` module Maven complains that it cannot resolve http client (2 http clients).

Reply:
Yeah, we'd have to specify the http client for every S3Client; using system property may be easier. 

Wrt the circular dependency issue, we probably can't default to url http client for IMDS client anyway because core depends on it and it will break customers once we add url http client in the classpath